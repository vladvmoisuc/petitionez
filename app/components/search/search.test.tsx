import { render } from "@testing-library/react";

import Search, { links } from ".";

const options = [{ title: "Test", slug: "test" }];

describe("<Search />", () => {
  it("should match the snapshot", () => {
    const { container } = render(<Search options={options} />);

    expect(container).toMatchSnapshot();
  });
});

describe("links", () => {
  it("should return the list of stylesheets", () => {
    expect(links()).toEqual([{ href: {}, rel: "stylesheet" }]);
  });
});
