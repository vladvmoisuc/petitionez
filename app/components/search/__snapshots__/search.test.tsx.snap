// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> should match the snapshot 1`] = `
<div>
  <div
    class="MuiAutocomplete-root MuiAutocomplete-hasPopupIcon Search css-u3zopq-MuiAutocomplete-root"
  >
    <div
      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedStart MuiAutocomplete-inputRoot css-l4u8b9-MuiInputBase-root-MuiInput-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-yurbhn-MuiSvgIcon-root"
          data-testid="SearchIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
          />
        </svg>
        <input
          aria-autocomplete="list"
          aria-expanded="false"
          aria-invalid="false"
          autocapitalize="none"
          autocomplete="off"
          class="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedStart MuiAutocomplete-input MuiAutocomplete-inputFocused css-1x51dt5-MuiInputBase-input-MuiInput-input"
          id=":r0:"
          placeholder="CautÄƒ o campanie"
          role="combobox"
          spellcheck="false"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;
